### PR TITLE
469 2x 4x without cache

### LINKED ISSUE #469: Add 2x and 4x speed without caching

### PR DESCRIPTION
**Pull request by hvangeffen**
Closes #469

### CODE REVIEW

**Review by vanmeeuwen — COMMENTED** [review]
Currently the implementation is that you need to press play and toggle a speed. It is more elegant to press either "play" or "double speed play" or "quadruple speed play".

File: src/components/DateTimeSlider.vue [hunk 1]
```diff
@@ -27,6 +27,16 @@
...
+          <v-btn :color="speedColor(speed.twoTimes)" icon @click="toggleSpeed(speed.twoTimes)" ref="TwoTimesButton">
+            <v-icon>
+              mdi-chevron-double-right
+            </v-icon>
+          </v-btn>
+          <v-btn :color="speedColor(speed.fourTimes)" icon @click="toggleSpeed(speed.fourTimes)" ref="FourTimesButton">
+            <v-icon>
+              mdi-chevron-triple-right
+            </v-icon>
+          </v-btn>
```
Comment (vanmeeuwen) [inline src/components/DateTimeSlider.vue:39]: Add these buttons immediately next to the play button.

**Review by vanmeeuwen — COMMENTED** [review]
We had replaced this.stopPlay() with this.index = 0
Were is this change now? I know it is not part of the current task, but I wonder what happened.

**Comment by hvangeffen** [comment]
> We had replaced this.stopPlay() with this.index = 0 Were is this change now? I know it is not part of the current task, but I wonder what happened.

This change is from a branch which I worked on previously, as such the new code has not been pulled in yet and since there are no conflicts merging is allowed.

**Pull request merged**